activerecord/lib/arel/table.rb

```diff
@@ -14,7 +14,8 @@ class << self; attr_accessor :engine; end
     alias :table_name :name
```

Comment:
Is this used anywhere internally? It sounds like any existing callers might be surprised to see not-a-table-name in there.

Reply:
Doesn't seem to be used anymore. Tests pass without it, dropped.